server/home/movieHelpers.js

```diff
@@ -0,0 +1,31 @@
+const { query } = require("express")
+
```

Comment:
in the future, it would be cool if we could use these helpers to do the same thing with tv shows. It'd be great if it were all in the same helper. I think the main difference between the two was movie uses "title" and tv shows use "name". Anyway, these look fabulous. Great work!

Reply:
great idea!  if we used graphql - we wouldnt even need to right helpers to return only the properties we needed.  We could litterally just query movieschema for what we needed and thats it